packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -76,15 +45,37 @@ export const styles = css`
             );
             border-color: rgba(${fillSelectedRgbPartialColor}, 0.3);
         }
-
-        :host([disabled]) .control[aria-pressed='true']:hover {
-            background-image: linear-gradient(
-                ${fillSelectedColor},
-                ${fillSelectedColor}
-            );
-            background-size: 100% 100%;
-            border-color: rgba(${fillSelectedRgbPartialColor}, 0.3);
-            box-shadow: none;
-        }
     }
-`;
+`.withBehaviors(
+    appearanceBehavior(
+        ButtonAppearance.outline,
+        css`
+            @layer base-selected {
```

Comment:
Need to override the `base-selected` styling for the outline-accent appearance, so that it keeps the same accent color for the label and icons, even when toggled-on.

Reply:
Do you need to actually add a new named layer? Can't we just use `base` again? it should make any selector in here win.

Reply:
Yes, if we just use the `base` layer, then the default "accent" styling takes precedence over the "pressed" styling because of specificity:
![image](https://github.com/ni/nimble/assets/7282195/821b0dcb-0cbd-4ac1-9513-77f6f134e467)
We want the "pressed" styling to always override the default styling regardless of specificity, which is what CSS layers are designed to do.